src/main/resources/templates/fragments/inputs/date.html

```diff
@@ -3,6 +3,7 @@
     th:with="
       hasLabel=${!#strings.isEmpty(label)},
       hasAriaLabel=${!#strings.isEmpty(ariaLabel)},
+      requireDateFields=${requireDateFields ?: false},
```

Comment:
Line 11 has a "required" variable - is that doing the same thing under a different name?

Reply:
Nope, just me totally not realizing I had already done the thing [😅](https://emojipedia.org/grinning-face-with-sweat)